src/autocapture-utils.js

```diff
@@ -190,6 +180,25 @@ export function shouldCaptureElement(el) {
     return true
 }
 
+/* 
+ * Check whether a DOM element is 'sensitive' and we should only capture limited data
+ * @param {Element} el - element to check
+ * @returns {boolean} whether the element should be captureed
+ */
+export function isSensitiveElement(el) {
```

Comment:
just an idea: Allow custom defined (in)sensibility `<input data-posthog-insensible="true" ...>` .

Simething like (not tested):

```
    if (el.dataset.posthogInsensible == "true" ) { return fase }
```

Reply:
Hm is there a specific usecase you're thinking of? Mixpanel got into a lot of trouble for messing this up and accidentally including passwords so I'm loath to add any complexity that isn't absolutely necessary

Reply:
It cannot be trouble of PostHog if user intentionally add attribute `data-posthog-insensible="true"` I think.

Particular usecase in my case is do a funnel of sing up, particularly, how many users drop off by not checking user agreement (clicking checkbox). Or another usecase: How many users use default distance (e.g. `500` in <`input type="number">). But it is mostly artificial usecases. I am new to business analysis I don't know what make sense to measure.

Reply:
I was curious as to what trouble mixpanel got into and found [this](https://techcrunch.com/2018/02/05/mixpanel-passwords/):

> We immediately began investigating further and learned that the behavior the customer was observing was due to a change to the React JavaScript library made in March 2017. This change placed copies of the values of hidden and password fields into the input elements’ attributes, which Autotrack then inadvertently received. Upon investigating further, we realized that, because of the way we had implemented Autotrack when it launched in August 2016, this could happen in other scenarios where browser plugins (such as the 1Password password manager) and website frameworks place sensitive data into form element attributes.

Ouch. So indeed we need to be careful with what we track.

I do see the value in optionally capturing an input field's `value` attribute though!